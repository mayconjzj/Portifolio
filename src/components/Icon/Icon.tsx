export const Icon = ({ icon }: { icon: React.ReactNode }) => {
  return <div className="text-2xl">{icon}</div>;
};
