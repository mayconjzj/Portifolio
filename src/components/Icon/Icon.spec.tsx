import { render } from '@testing-library/react';

import { Icon } from './Icon';

describe('Icon', () => {
  it('should render the icon', () => {
    const { container } = render(<Icon>Next test</Icon>);
    expect(container).toMatchSnapshot();
  });
});
