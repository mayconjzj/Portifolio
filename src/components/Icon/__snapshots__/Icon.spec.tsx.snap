// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render the icon 1`] = `
<div>
  <div
    class="text-2xl"
  >
    Next test
  </div>
</div>
`;
